app/views/govuk_publishing_components/components/docs/layout_for_admin.yml

```diff
@@ -10,7 +10,11 @@ body: |
 display_preview: false
 display_html: true
 accessibility_criteria: |
-  TBD
+  The layout template must:
+
+    * have a `lang` attribute
+    * contain a [skip link](/component-guide/skip_link)
```

Comment:
Viewing this branch locally, the skip link when focussed looks different to how it does on the live site. Is this deliberate or an unexpected consequence of the preview? Live site:

<img width="988" alt="Screenshot 2020-07-24 at 15 01 37" src="https://user-images.githubusercontent.com/861310/88399312-9bd30680-cdbe-11ea-9ad6-f479f011be00.png">

PR:

<img width="1440" alt="Screenshot 2020-07-24 at 15 00 55" src="https://user-images.githubusercontent.com/861310/88399336-a2fa1480-cdbe-11ea-9688-3d095c5872ac.png">

Reply:
Having looked into it no, I don't think this is an unintended consequence of the preview, but rather a consequence of replacing the old skip link with [the Design System one](https://design-system.service.gov.uk/components/skip-link/default/index.html) which has different styling.

The skip link on the has live site some extra wrappers with styling from [govuk_template CSS](https://github.com/alphagov/govuk_template/blob/134e8b910c5175d5f94b1f5c48fdc342bb5065c2/source/assets/stylesheets/_accessibility.scss#L85).

I'd be keen to stick with the DS styling (I actually prefer it to what we've got on the live site currently 🤷🏻‍♀️) but let me check with a designer and get back to you on that.

Reply:
@andysellick I have discussed this with @miaallers and she's happy to go with the Design System styling for this (can't see a reason why not).

Reply:
Good response. I have only one remaining question - is it supposed to have that thin black line near the bottom of the highlighted yellow?

Reply:
@andysellick I think this is a known Chrome bug –  I believe it was fixed in the Design System here https://github.com/alphagov/govuk-frontend/pull/1879/files

Reply:
If it was fixed in the Design system can it be fixed here too?

Reply:
@andysellick Is it not fixed for you? 
If you can still see the outdated styling on your local version, it would probably help if you run `yarn install` in the project. This should update `govuk-frontend` to the latest version  (it should be 3.8.0).

Reply:
Ah yes that seems to have done it, thanks.